mac

```diff
@@ -105,8 +112,7 @@ deactivate_exit_on_error() {
 
 install_homebrew() {
   fancy_echo "Installing Homebrew ..."
-  curl -fsS \
-    'https://raw.githubusercontent.com/Homebrew/install/master/install' | ruby
+  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
The installation process of `homebrew` changed, this is the new [recommend command](https://docs.brew.sh/Installation)